accounts/tasks.py

```diff
@@ -38,7 +38,7 @@ def update_portfolio_value():
         portfolio_value = user.current_portfolio_value()
         if user.portfolio_value != portfolio_value:
             user.portfolio_value = portfolio_value
-            user.save(update_fields=['portfolio_value'])
+            user.save()
```

Comment:
This is the only way to update reputation I think. Do you think that
signals are better?
8 kwi 2016 23:19 "Wojciech Zając" <notifications@github.com> napisał(a):

> In accounts/tasks.py
> <https://github.com/KlubJagiellonski/Politikon/pull/131#discussion_r59090791>
> :
>
> > @@ -38,7 +38,7 @@ def update_portfolio_value():
> >          portfolio_value = user.current_portfolio_value()
> >          if user.portfolio_value != portfolio_value:
> >              user.portfolio_value = portfolio_value
> > -            user.save(update_fields=['portfolio_value'])
> > +            user.save()
>
> Are you sure is it safe? You want to save all user values asynchronously
> in task?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/KlubJagiellonski/Politikon/pull/131/files/d40fa5b58d95432d7b53878c4e997987225e563c#r59090791>
>